src/Symfony/Component/Form/Extension/Core/DataTransformer/DateTimeImmutableToDateTimeTransformer.php

```diff
@@ -30,7 +30,7 @@ final class DateTimeImmutableToDateTimeTransformer implements DataTransformerInt
      *
      * @throws TransformationFailedException If the given value is not a \DateTimeImmutable
      */
-    public function transform($value)
+    public function transform($value): ?\DateTime
```

Comment:
You can remove the corresponding `@return` annotation.

Reply:
I have not removed the annotation like `@return AwesomeType[]` or `@return AwesomeType an awesome instance`, as I don't really know where the cursor between usefull, or useless annotaiton is. @nicolas-grekas any thoughts?